x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/timeline_entry.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import moment from 'moment';
+
+import { EuiAvatar, EuiComment, EuiText } from '@elastic/eui';
+import { Immutable, EndpointAction } from '../../../../../../../common/endpoint/types';
+
+export const TimelineEntry = memo(
```

Comment:
Not for this PR.

The naming "timeline" might get confusing in our code based because we actually have an entire section of security solution named `timeline`. My suggestion would be to rename anything with `timeline` to a more specific name - like `log_entry`?

Reply:
That makes sense 👍  I'd change that.

Reply:
done 31cf7af4edb6471cf953c2df1fdeea5cfd61f65a